src/PrestaShopBundle/Twig/Component/HeadTag.php

```diff
@@ -134,7 +138,7 @@ public function getShopName(): string
 
     public function getControllerName(): string
     {
-        return $this->menuBuilder->getLegacyControllerClassName() ?: '';
+        return $this->legacyControllerContext->controller_name;
     }
 
     public function getImgDir(): string
```

Comment:
You can get the baseUrl from `ShopContext` I believe no?

Reply:
Yes, now it's possible